tests/contrib/hooks/test_gcp_vision_hook.py

```diff
@@ -102,50 +113,48 @@ def test_create_productset_autogenerated_id_wrong_api_response(self, get_conn):
         create_product_set_method = get_conn.return_value.create_product_set
         create_product_set_method.return_value = response_product_set
         parent = ProductSearchClient.location_path(PROJECT_ID_TEST, LOC_ID_TEST)
-        hook = self.vision_hook_default_project_id
         product_set = ProductSet()
         # When
         with self.assertRaises(AirflowException) as cm:
-            hook.create_product_set(location=LOC_ID_TEST, product_set_id=None, product_set=product_set,
-                                    project_id=PROJECT_ID_TEST, retry=None, timeout=None, metadata=None)
+            self.hook.create_product_set(location=LOC_ID_TEST, product_set_id=None, product_set=product_set,
+                                         project_id=PROJECT_ID_TEST, retry=None, timeout=None, metadata=None)
         # Then
         # API response was wrong (None) and thus ProductSet ID extraction should fail.
         err = cm.exception
         self.assertIn('Unable to get name from response...', str(err))
         create_product_set_method.assert_called_once_with(parent=parent, product_set=product_set,
                                                           product_set_id=None, retry=None, timeout=None,
                                                           metadata=None)
-
+    # get_productset
```

Comment:
I think we should split this Hook test into separate classes rather than having those comments.

Reply:
I removed a comment. I do not want to make big changes not related to my patch. We have other operators to do, and complex PRs will make reviews difficult.